configure

```diff
@@ -584,10 +584,10 @@ MAKEFLAGS=
 PACKAGE_NAME='PipelineDB'
 PACKAGE_TARNAME='pipelinedb'
 PACKAGE_VERSION='9.4.4'
-PACKAGE_STRING='PostgreSQL 9.4.4'
+PACKAGE_STRING='PipelineDB 9.4.4'
```

Comment:
Changing these constants is actually pretty sensitive and will cause issues with our binary builds. If you want to keep this change in, we'll want to run all the buildbot builds against this change first.